content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -1,79 +1,121 @@
-Configuration --> Settings --> Product Catalog` and activate one or both of the
-:guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings. Finally, click
-:guilabel:`Save` to apply the settings.
+.. note::
+   It should be noted that the setting is titled, `Variant Grid Entry` on the *Sales* app settings
+   page, and titled, `Order Grid Entry` on product forms. So, be sure to keep that in mind.
+
+Settings
+========
+
+When working with product variants, Odoo uses the :guilabel:`Product Configurator`, by default. To
+add the :guilabel:`Variant Grid Entry` option, that feature **must** be enabled in the Odoo *Sales*
+application.
 
-.. image:: orders_and_variants/activating-entry-type-settings.png
+To enable that setting, go to :menuselection:`Sales app --> Configuration --> Settings`, and scroll
+to the :guilabel:`Product Catalog` section. Then, check the box next to the :guilabel:`Variant Grid
+Entry` option, and click :guilabel:`Save`.
+
+.. image:: orders_and_variants/order-grid-entry-setting.png
    :align: center
-   :alt: Activating entry type settings.
+   :alt: The variant grid entry setting in the Odoo Sales application
+
+.. note::
+   Of course, the :guilabel:`Variants` feature **must** also be activated, as well, in order to use
+   product variants on quotations and sales orders.
+
+Product configuration
+=====================
+
+Once the :guilabel:`Variant Grid Entry` setting is enabled, both options (:guilabel:`Product
+Configurator` and :guilabel:`Order Grid Entry`) become available on every product form.
+
+To configure a product form to use either a product configurator or variant grid entry, start by
+navigating to :menuselection:`Sales app --> Products --> Products` to view all the products in the
+database.
+
+Then, select the desired product to configure, or click :guilabel:`New` to create a new product from
+scratch. Once on the product form, click into the :guilabel:`Attributes & Variants` tab, where
+product variants can be viewed, modified, and added.
 
-After saving the settings, configure the product form. Head over to :menuselection:`Sales -->
-Products --> Products` and select any product (or click :guilabel:`Create` to create a new one).
-On the product form, click :guilabel:`Edit`, and then click on the :guilabel:`Variants` tab. Under
-:guilabel:`Sales Variant Selection`, choose either :guilabel:`Product Configurator` or
-:guilabel:`Order Grid Entry`. This selection determines which method is used to add the product to
-quotations or sales orders. Finally, click :guilabel:`Save` to apply the setting.
+At the bottom of the :guilabel:`Attributes & Variants` tab, there is a :guilabel:`Sales Variant
+Selection` section with two options: :guilabel:`Product Configurator` and :guilabel:`Order Grid
+Entry`.
 
-.. image:: orders_and_variants/sales-variant-selection.png
+.. note::
+   It should be noted that these options **only** appear if attributes have been added to the
+   record.
```

Comment:
To add to Jess' earlier comment, I tested adding 2 attributes with 1 value each, and the two Sales  Variant Selection options were still not available
![image](https://github.com/odoo/documentation/assets/23611711/444c1c5a-432a-4b3f-a14d-dee422e05261)

```suggestion
.. note::
   It should be noted that these options **only** appear if at least two values of an attribute have
   been added to the record.
```